css/secondary-page.styl

```diff
@@ -40,6 +40,16 @@
           font-weight: 400
           margin-bottom: 1em
 
+          img.avatar
+            padding-right: 10px !important
+
+          span.login-name
+            padding-left: 7px !important
+
+            span
+              font-size: 0.65em !important
+              font-weight: 200 !important
+
```

Comment:
Could these be scoped within css where they're being used and use clear class names rather than tag? `!important` should be avoided and they don't look necessary here anyway. Also, you can apply the `font-size` and `font-weight` to `login-name` itself.